src/main/java/com/lpvs/service/LPVSQueueProcessorService.java

```diff
@@ -51,12 +57,13 @@ public class LPVSQueueProcessorService {
      * @throws Exception If an exception occurs during queue processing.
      */
     @EventListener(ApplicationReadyEvent.class)
-    private void queueProcessor() throws Exception {
+    protected void queueProcessor() throws Exception {
         // Check for any pending elements in the LPVSQueue.
         queueService.checkForQueue();
 
         // Process LPVSQueue elements until the trigger is set.
-        while (trigger == null || trigger.isEmpty()) {
+        while ((trigger == null || trigger.isBlank())
+                && (localPath == null || localPath.isBlank())) {
```

Comment:
`localPath.isBlank()` includes case with null, isn't it?
If so, there is no reason to check for `localPath == null`